src/coreclr/debug/daccess/dacdbiimpl.cpp

```diff
@@ -4485,6 +4485,11 @@ void  DacDbiInterfaceImpl::EnumerateAssembliesInAppDomain(
     // See comment in code:DacDbiInterfaceImpl::EnumerateModulesInAssembly code for details.
     AppDomain * pAppDomain = vmAppDomain.GetDacPtr();
 
+    if (pAppDomain == nullptr)
```

Comment:
What is the scenario where this would occur? Is this done via an SOS command or is there some logging other tooling that makes this call at a bad time? I'm simply unsure how one could enumerate the assemblies in the AppDomain prior to the AppDomain being created. This confusion is basically around the user passing in `VMPTR_AppDomain`, if that is invalid, why/how did they get it?

Reply:
Had an offline discussion with @AaronRobinsonMSFT, there is no good top-level place to handle this, we will leave it as is.